include/aktualizr-lite/api.h

```diff
@@ -51,6 +52,9 @@ class CheckInResult {
   // NOLINTNEXTLINE(hicpp-explicit-conversions,google-explicit-constructor)
   operator bool() const { return status == CheckInResult::Status::Ok || status == CheckInResult::Status::OkCached; }
 
+  TufTarget selected_target = TufTarget();
```

Comment:
I would introduce a new type and use it as a return type of `GetTargetToInstall`.
Extending the `CheckInResult` with these two fields looks like mixing fields/data of two different entities in one class.

Reply:
Yep, makes sense. I was avoiding the creation of new types, by I agree that having these new fields might bring confusion to users of the original types.

Reply:
I've created a new GetTargetToInstallResult with the new fields, and its own Status enum.